conda/core/path_actions.py

```diff
@@ -13,7 +13,7 @@
 from .prefix_data import PrefixData
 from .._vendor.auxlib.compat import with_metaclass
 from .._vendor.auxlib.ish import dals
-from ..base.constants import CONDA_TARBALL_EXTENSION
+from ..base.constants import CONDA_TARBALL_EXTENSION, SafetyChecks
```

Comment:
This import surprises me. I am in favor right now of moving the `SafetyChecks.disabled` logic from `conda/core/link.py` to `conda/core/path_actions.py`.  There may be several `PathActions` that have to be adjusted.